### PR TITLE
Adding a new container flavour for Aquasecurity's Cloudsploit

### PR DESCRIPTION
**Pull request by axelpavageau**

### CODE REVIEW

File: cloudsploit/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+LABEL maintainer="Axel Pavageau <axel.pavageau@ekino.com>"
+
+ADD package.json .
+
+RUN apt-get update -qq \
+    && pip install -U pip \
+    && pip install pipenv awscli boto3 \
+    && apt install -y npm git wget zip
+
+RUN npm install \
```
Comment (mremi) [inline cloudsploit/Dockerfile:12]:
```
npm install -g cloudsploit@^2.0
```
does not work?

Comment (Crow-EH) [reply cloudsploit/Dockerfile:12]: I think @axelpavageau did that to eventually let dependabot scan package.json and upgrade the dependency automatically.

If so, how about setting it up now @axelpavageau ?

Comment (axelpavageau) [reply cloudsploit/Dockerfile:12]: @Crow-EH is right, this is setup for dependabot to handle (although I haven't tested whether it's able to handle this specific dependency git/tag format).

I've added the missing dependabot configurations

File: .github/dependabot.yml [hunk 2]
```diff
@@ -98,6 +98,29 @@ updates:
...
+    directory: "/cloudsploit"
+    schedule:
+      interval: "weekly"
+    labels:
+      - "dependencies"
+      - "npm"
+      - "cloudsploit"
+    ignore:
+      - dependency-name: "*"
+        update-types: ["version-update:semver-major"]
```
Comment (Crow-EH) [inline .github/dependabot.yml:123]: shouldn't it be `semver-minor` since you put version to `2` in `cloudsploit/config.yml` ?

Comment (axelpavageau) [reply .github/dependabot.yml:123]: it's an ignore statement, so we'll only get minor & patches

Comment (Crow-EH) [reply .github/dependabot.yml:123]: Oooh my bad. LGTM then

**Review by Crow-EH — APPROVED** [review]

**Comment by Crow-EH** [comment]
> although I haven't tested whether it's able to handle this specific dependency git/tag format
It should work.
We'll be able to check if it does after merging  & dependabot's next checks : https://github.com/ekino/docker-buildbox/network/updates

**Pull request merged**